app/src/main/java/com/strv/movies/ui/login/MoviesLoginScreen.kt

```diff
@@ -0,0 +1,188 @@
+    }
+}
+
+@Composable
+private fun PasswordField() {
+    val passwordState = remember { mutableStateOf(TextFieldValue()) }
+    var passwordVisible by rememberSaveable { mutableStateOf(false) }
+
+    TextField(
+        value = passwordState.value,
+        onValueChange = { passwordState.value = it },
+        modifier = Modifier.padding(bottom = 10.dp),
+        isError = false,
+        placeholder = { Text("Password") },
+        label = {
+            Text(text = stringResource(id = R.string.login_password_text_input_field))
+        },
+
+        visualTransformation = if (passwordVisible) VisualTransformation.None else PasswordVisualTransformation(),
+        keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Password),
+        trailingIcon = {
+            val image = if (passwordVisible)
+                Icons.Filled.Visibility
+            else Icons.Filled.VisibilityOff
+
+            // Please provide localized description for accessibility services
+            val description = if (passwordVisible) "Hide password" else "Show password"
+
+            IconButton(onClick = { passwordVisible = !passwordVisible }) {
+                Icon(imageVector = image, description)
+            }
+        }
+    )
+}
+
+@Composable
+private fun EmailField() {
+    val emailState = remember { mutableStateOf(TextFieldValue()) }
+    TextField(
+        value = emailState.value,
+        onValueChange = { emailState.value = it },
+        modifier = Modifier.padding(bottom = 10.dp),
+        isError = false,
+        label = {
+            Text(
+                text = stringResource(id = R.string.login_username_text_input_field)
+            )
+        }
+    )
+}
+
+@Composable
+private fun ContinueWithFacebookButton(navigateToMovieList: () -> Unit) {
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier
+            .fillMaxWidth()
+    ) {
+        Button(
+            onClick = { navigateToMovieList() },
+//            onClick = { "https://www.facebook.com/index.php" },
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(bottom = 10.dp)
+        ) {
+            Text(
+                modifier = Modifier.padding(start = 10.dp, end = 10.dp),
+                text = stringResource(id = R.string.continue_with_fb)
+            )
+            Icon(
+                painter = painterResource(R.drawable.ic_facebook),
+                contentDescription = stringResource(id = R.string.continue_with_fb),
+                modifier = Modifier.size(24.dp)
+            )
+        }
+    }
+    SignInGmailButton(navigateToMovieList)
+}
+
+@Composable
+private fun SignInGmailButton(navigateToMovieList: () -> Unit) {
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier
+            .fillMaxWidth()
+    ) {
+        Button(
+            onClick = { navigateToMovieList() },
```

Comment:
Just a hint: you can directly assign the lambda here like `onClick = navigateToMovieList` because the onClick and navigateToMovieList are lambdas with the same type `() -> Unit`

Reply:
I don't thinks it's a good idea ...
![Screenshot 2022-05-20 224250](https://user-images.githubusercontent.com/36711021/169607868-311fa3ee-3379-404c-943b-ce4d294edded.jpg)

Reply:
No, you put there the type of lambda. What I meant is that you could remove just the braces in this case like `Button(onClick = navigateToMovieList, ...)`